7Cav-Alive.Altis/scripts/client/clearItemsBASE_switch.sqf

```diff
@@ -0,0 +1,5 @@
+CLIENT_BaseCleanup = {

```

Comment:
Have you updated the `mission.sqm` to call this function instead of calling this file?

Reply:
Yes, there is no longer an interaction in the mission anymore since the loop gets initialized at mission start.